src/js/components/RangeInput/StyledRangeInput.js

```diff
@@ -1,14 +1,75 @@
+import { rgba } from 'polished';
 import styled, { css } from 'styled-components';
 
 import { focusStyle, normalizeColor, parseMetricToNum } from '../../utils';
 import { defaultProps } from '../../default-props';
 
+// opacity of the bound trumps the track opacity
+const getBoundOpacity = (props, bound) => {
+  return props.theme.rangeInput &&
+    props.theme.rangeInput.track &&
+    props.theme.rangeInput.track[bound] &&
+    props.theme.rangeInput.track[bound].opacity
+    ? props.theme.rangeInput.track[bound].opacity
+    : 1;
+};
+
+const getBoundColor = (props, bound) => {
+  if (
+    props.theme.rangeInput &&
+    props.theme.rangeInput.track &&
+    props.theme.rangeInput.track[bound] &&
+    props.theme.rangeInput.track[bound].color
+  ) {
+    return rgba(
+      normalizeColor(props.theme.rangeInput.track[bound].color, props.theme),
+      getBoundOpacity(props, bound),
+    );
+  }
+  // If bound color is undefined pick the default track color with bound opacity
+  return rgba(
+    normalizeColor(props.theme.rangeInput.track.color, props.theme),
+    getBoundOpacity(props, bound),
+  );
+};
+
+const trackColorStyle = props => {
+  // backward compatibility in case no bounds are defined
+  if (
+    props.theme.rangeInput &&
+    props.theme.rangeInput.track &&
+    !props.theme.rangeInput.track.lower &&
+    !props.theme.rangeInput.track.upper
+  ) {
+    const color = rgba(
+      normalizeColor(props.theme.rangeInput.track.color, props.theme),
+      0.2,
```

Comment:
I think we should be pulling the opacity from `props.theme.rangeInput.track.opacity`.

Reply:
I agree, but not on this section, I added a block below that one that will use `props.theme.rangeInput.track.opacity` in case the user changed the color and wanted to use the theme opacity. 
This block is to maintain backward compatibility.

Reply:
But, from line 884 in base.js we have `// opacity = undefined`. If I set `theme.rangeInput.track.opacity = 0.4` in my theme, isn't this where it would be used?

Reply:
It will be used in line #53.